ui/desktop/electron-app/src/utils/jsonify.js

```diff
@@ -11,7 +11,6 @@ const jsonify = (data) => {
   try {
     return JSON.parse(data);
   } catch (e) {
-    log.warn('jsonify:', e);
```

Comment:
None blocking comment: we delete the entire log, is this what we want? or do we want to log `e.message` rathern than `e`?

Reply:
It's not a useful log statement so I added log messages where from the callers of this method instead